bib/stable.bib

```diff
@@ -49651,6 +49652,8 @@ @book{Hare1981MoralThinkingIts
 	langid = {english},
 	title = {Moral thinking: Its levels, method, and point},
 	isbn = {0-19-824660-9},
+	doi = {10.1093/0198246609.001.0001},
```

Comment:
unsure how you prefer things -- the `doi` when you go to the doi.org/{doi} url will take you to the address I've added as `url` just below. Unsure if you would like both or just one of these 👍

Reply:
We give precedence to the DOI when available, since (I think) it is less susceptible to linkrot, and is also more handy for finding the associated PDF.

Reply:
I like the idea -- I'll aim to do the same over at https://github.com/whyboris/utilitarianism.net on the Spanish translation of the site you worked on 👍